test/conftest.py

```diff
@@ -70,12 +70,22 @@ def image_list(request, account, region):
 
 @pytest.fixture(scope='session')
 def use_local_jobs(request):
-    return request.config.getoption('--use-local-jobs')
+    value = request.config.getoption('--use-local-jobs')
```

Comment:
Why not use the default value here? Same for all the other `if value is None` lines.